src/main/kotlin/racingcar/domain/RacingCarGame.kt

```diff
@@ -22,4 +22,29 @@ class RacingCarGame(
         currentNumber++
         return true
     }
+
+    fun winners(): String {
+        val maxPosition = maxPosition()
+        return createWinners(maxPosition)
+    }
+
+    private fun maxPosition(): Int {
+        var maxPosition = 0;
+        racingCars
+            .asSequence()
+            .filter { it.position() > maxPosition }
+            .forEach { maxPosition = it.position() }
+
+        return maxPosition;
+    }
+
+    private fun createWinners(maxPosition: Int): String {
+        val winners = mutableListOf<String>()
+        racingCars
+            .asSequence()
+            .filter { it.position() == maxPosition }
+            .forEach { winners.add(it.name()) }
+
+        return winners.joinToString { it }
+    }
```

Comment:
```suggestion
        private fun createWinners(maxPosition: Int): String =
        racingCars
            .filter { it.position() == maxPosition }        
            .joinToString { it.name() }
```

이렇게도 가능하겠죠? 추가로 asSequence를 사용한 이유가 있으실까요? 현재 로직으로써는 제외해도 괜찮을 것 같아서요🤔

Reply:
없어도 문제가 없는 로직이군요!